src/index.js

```diff
@@ -0,0 +1,144 @@
+const GARDENSKIES = {
+  sunny: '☁️ ☁️ ☁️ ☀️ ☁️ ☁️',
+  cloudy: '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️',
+  rainy: '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧',
+  snowy: '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨',
+};
+
+const CONDITIONS = {
+  hot: { landscape: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂', text: 'red' },
+  warm: { landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷', text: 'orange' },
+  moderate: { landscape: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃', text: 'yellow' },
+  chilly: { landscape: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲', text: 'green' },
+  cold: { landscape: '❄️🌲⛄️🌲❄️🏂⛄️🌲❄️⛷🌲❄️🌲', text: 'teal' },
+};
+
+const state = {
+  tempValue: 60,
+  cityLow: null,
+  cityHigh: null,
+  weatherDescription: null,
+};
+
+const kelvinToFahrenheit = (temperature) =>
+  (temperature - 273.15) * (9 / 5) + 32;
+
+const updateTemp = () => {
+  const tempValue = document.querySelector('#tempValue');
+  tempValue.textContent = state.tempValue;
+  conditionLayout(state.tempValue, tempValue);
+};
+
+const incrementTemp = () => {
+  state.tempValue += 1;
+  updateTemp();
+};
+
+const decrementTemp = () => {
+  state.tempValue -= 1;
+  updateTemp();
+};
+
+const conditionLayout = (temp, el) => {
+  const landscapeLayout = document.querySelector('#gardenLandscape');
+  let currentCondition = null;
+  if (temp >= 80) {
+    currentCondition = CONDITIONS.hot;
+  } else if (temp < 80 && temp >= 70) {
+    currentCondition = CONDITIONS.warm;
+  } else if (temp < 70 && temp >= 60) {
+    currentCondition = CONDITIONS.moderate;
+  } else if (temp < 60 && temp >= 50) {
+    currentCondition = CONDITIONS.chilly;
+  } else if (temp < 50) {
+    currentCondition = CONDITIONS.cold;
+  }
+  el.style.color = currentCondition.text;
```

Comment:
Using class names for this would have put all the styling in one location, though admittedly having your settings at the top of the file makes that less necessary.